Bug 930871 - Fix CDMA data connection terminology term R=arthurcc

Aspirar opened the pull request:
I have made the changes in apps/settings/js/carrier_iccs.js

Commit:
Bug 1102340 - + symbol is shown on the wrong side of phone numbers

Commit:
Bug 910552 - [Settings] Add Call Barring menu (r=arthurcc)

ochameau committed:
Bug 1043953 - Set `origin` field in manifest.webapp files. r=rickychien

steveck-chung committed:
Bug 1118154 - [FFOS7715 v2.1] [dolphin] FFOS can not receive Cell Broadcast messages on SIM2, r=kgrandon

Commit:
Bug 1115644 - Implement GeneralPanel and standardize panel interface for KeyboardSettingsApp

- Implement a ViewBase which is the base class of all views and panels. It defines an API for view transitioning and child views.
- PanelController is always included during building.
- KBSettingsApp no longer touches views directly; knowledge of panels/dialogs is moved to PanelController/DialogController from KBSettingsApp.
- GeneralPanel is the "root" panel, renamed as such; in PanelController, the "root" panel may be dynamically designated, with default being GeneralPanel.
- Change init/uninit verbs to start/stop in panels/dialogs. Also, start() must now be explicitly called for symmetry.
- Rename view js filenames to _view.js .

steveck-chung committed:
Bug 1091511 - [Message]The number/email or URL can't be tapped if there is full-width text in this message, r=azasypkin

stasm committed:
Bug 1124098 - Hide the Get More Languages link in devices other than phones. r=arthurcc

Commit:
Bug 1123596 - [Stingray] Receive remote notifications with Presentation API and display them

mancas committed:
Bug 1118854 - [Window Management][Homescreen] If the user taps home while opening task manager, the homescreen will appear blank. r=sfoster

jrburke committed:
Bug 1124081 - Warning about configuring email without SSL has not enough space for localization

ochameau committed:
Bug 1043953 - Add package.json for each app to educate WebIDE about how to build gaia apps. r=rickychien

Aspirar committed:
Bug 930871 - Fix CDMA data connection terminology term R=arthurcc

Aspirar committed:
Bug 930871 - Fix CDMA data connection terminology term R=arthurcc

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/6514847?v=3" alt="Aspirar" width="50px" height="50px" /> Lakshya Gupta (Aspirar)](https://github.com/Aspirar) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8a0455445024)

Commit:
Bug 1112048 - Update apn list. r=jaoo

phoenix1moz committed:
Bug 1113662 - [v2.2] Investigate test_sms_with_attachments tap_capture failure

stasm committed:
Bug 1125177 - Amend ctx.availableLocales when additionallanguageschange event is fired. r=gandalf

etiennesegonzac committed:
Bug 1115027 - Making sure the identification-overlay appears instantly and only fades-out r=cwiiis

huchengtw-moz committed:
Bug 1123300 - [Stingray] use querySelectorAll to query buttons

ash7594 committed:
Bug 1125525 - Overflow of contact name while tapping on the recipient in composer r=pivanov r=julien

huchengtw-moz committed:
Merge pull request #27516 from huchengtw-moz/bug-1123300-context-menu-not-shown

Bug 1123300 - [Stingray] use querySelectorAll to query buttons, r=rexboy

KevinGrandon committed:
Bug 1125696 - Disable intermittent failing test, TEST-UNEXPECTED-FAIL | apps/verticalhome/test/marionette/group_collapse_drag_into_test.js | Vertical - Group check dragging icon into collapsed group

Commit:
Bug 1125703 - Disable Gaia Keyboard Build Integration Tests until bug 1119731 is fixed

- Disabled tests of GAIA_KEYBOARD_{PRELOAD_DICT_}LAYOUTS=* and GAIA_KEYBOARD_LAYOUTS=noPreloadDictRequired

Commit:
Bug 1125704 - Prevent notification from exceeding screen width

begeeben committed:
Bug 1123262 - [Midori 2.0]WLAN:signal strenght levels are not supported on notification bar

* Correct the wifi 'onconnectioninfoupdate' usage.

shamenchens committed:
Bug 1094632 - Email local draft not showing attachment icon

rvandermeulen committed:
Merge pull request #27490 from mancas/bug1118854

Bug 1118854 - [Window Management][Homescreen] If the user taps home whil...

rvandermeulen committed:
Merge pull request #27660 from mnjul/bug_1125703_disable_kb_build_int_test

Bug 1125703 - Disable Gaia Keyboard Build Integration Tests until bug 1119731 is fixed

Commit:
Bug 1039186 - Keyboard's Arabic layout needs to be refactored

seinlin committed:
Merge pull request #27668 from begeeben/1123262_wlan_signal

Bug 1123262 - [Midori 2.0]WLAN:signal strenght levels are not supported ...

mozilla-autolander-deprecated committed:
Bug 1115027 - merge pull request #27639 from etiennesegonzac:bug-1115027 to mozilla-b2g:master

rvandermeulen committed:
Merge pull request #27630 from acperez/bug-1112048

Bug 1112048 - Update apn list. r=jaoo +autoland

jrburke committed:
Bug 1110532 - [Email][Search] Inputting then quickly deleting characters in the Email Search field causes text overlap with erroneous search results

mozilla-autolander-deprecated committed:
Bug 1102340 - merge pull request #27061 from anefzaoui:bug-1102340 to mozilla-b2g:master

jrburke committed:
Merge pull request #27688 from jrburke/bug1110532-email-empty-search

Bug 1110532 - [Email][Search] Inputting then quickly deleting characters in the Email Search field causes text overlap with erroneous search results r=asuth

jrburke committed:
Merge pull request #27576 from jrburke/bug1124081-email-ssl-warn-l10n-style

Bug 1124081 - Warning about configuring email without SSL has not enough space for localization r=asuth

KevinGrandon committed:
Bug 1126012 - Disable failing demo-keyboard unit tests

steveck-chung committed:
Merge pull request #27663 from steveck-chung/cb-dsds

Bug 1118154 - [FFOS7715 v2.1] [dolphin] FFOS can not receive Cell Broadcast messages on SIM2, r=kgrandon

dwi2 committed:
Merge pull request #27666 from dwi2/bug1125704

Bug 1125704 - Prevent notification from exceeding screen width, r=johnhu

dwi2 committed:
Merge pull request #27619 from dwi2/bug1123596

Bug 1123596 - [Stingray] Receive remote notifications with Presentation ...

Commit:
Merge pull request #27361 from mnjul/bug_1115644_kb_settings_rootpanel

Bug 1115644 - Implement RootPanel and standardize panel interface for KeyboardSettingsApp. r=timdream

KevinGrandon committed:
Merge pull request #27692 from KevinGrandon/bug_1126012_demo_keyboard_tests

Bug 1126012 - Disable failing demo-keyboard unit tests

stasm committed:
Merge pull request #27634 from stasm/1125177-register-additional

Bug 1125177 - Amend ctx.availableLocales when additionallanguageschange fires. r=gandalf

stasm committed:
Merge pull request #27627 from stasm/1124098-langpacks-hide-link

Bug 1124098 - Hide the Get More Languages link in devices other than phones. r=arthurcc

mozilla-autolander-deprecated committed:
Bug 1039186 - merge pull request #27684 from anefzaoui:bug-1039186 to mozilla-b2g:master

alastor0325 committed:
Bug 1125717 - [B2G] Add audio channel attribute to FM app r=baku

steveck-chung committed:
Merge pull request #27530 from steveck-chung/message-link-helper-safe-start

Bug 1091511 - [Message]Fix number/email URL can't be tapped if there is full-width text in this message, r=azasypkin

fcampo committed:
Merge pull request #27081 from fcampo/call-barring

Bug 910552 - [Settings] Call Barring menu not implemented (r=arthurcc)

asutherland committed:
Merge pull request #27675 from shamenchens/Bug1094632-EmailLocalDraft-master

Bug 1094632 - Email local draft not showing attachment icon. r=asuth

land https://github.com/mozilla-b2g/gaia-email-libs-and-more/pull/363

ochameau committed:
Merge pull request #27201 from ochameau/origin

Bug 1043953 - Set `origin` field in manifest.webapp files. r=rickychient

ochameau committed:
Merge pull request #27160 from ochameau/packageJson

Bug 1043953 - WebIDE should be able to build'n push gaia apps. r=rickychien

JohanLorenzo committed:
Merge pull request #27115 from viorelaioia/bug_1113662

Bug 1113662 - [v2.2] Investigate test_sms_with_attachments tap_capture f...

rvandermeulen committed:
Merge pull request #27701 from alastor0325/bug1125717

Bug 1125717 - [B2G] Add audio channel attribute to FM app r=baku

rvandermeulen committed:
Merge pull request #27658 from bzbarsky/fix-nonconfigurable-defineProperty

 Bug 1126124 - Make defineProperty calls on windows not try to create non-configurable properties in gaia. r=arthurcc,rlu

Aspirar committed:
Merge branch 'cdma-fix' of https://github.com/Aspirar/gaia into cdma-fix

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/6514847?v=3" alt="Aspirar" width="50px" height="50px" /> Lakshya Gupta (Aspirar)](https://github.com/Aspirar) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2f2e67ae76ee)

Aspirar committed:
Update telephony_items_handler.js

Fix CDMA data terminology

try-server-hook commented:
[<img src="https://avatars.githubusercontent.com/u/6514847?v=3" alt="Aspirar" width="50px" height="50px" /> Lakshya Gupta (Aspirar)](https://github.com/Aspirar) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=40a72615cc6d)
